src/site/content/en/learn/learn.11tydata.js

```diff
@@ -37,7 +37,6 @@ module.exports = function () {
     allPaths['lighthouse-performance'],
     allPaths['lighthouse-pwa'],
     allPaths['lighthouse-best-practices'],
-    allPaths['lighthouse-accessibility'],
```

Comment:
this removes `lighthouse-accessibility` from the https://web.dev/learn/ Lighthouse collection. I think this is the right thing to do? I assume when I complete the migration we just won't have a Lighthouse collection under learn/ at all

cc @rachelandrew if that's something you've thought about